rtos/TARGET_CORTEX/SysTimer.cpp

```diff
@@ -20,12 +20,23 @@
  * SOFTWARE.
  */
 #include "rtos/TARGET_CORTEX/SysTimer.h"
+#include "cmsis/TARGET_CORTEX_A/irq_ctrl.h"
```

Comment:
this header file is not available for all cortex thus should be protected somehow. Thinking, isn't this irq_ctrl.h available via RTE_Components or CMSIS_device_header (in our case is it cmsis.h file) ?

I assume the same lines as in rtos/TARGET_CORTEX/mbed_rtx_idle.cpp would do the job here

Reply:
This should be fixed now.